docs/tldr/pages/linux/tmt-run.md

```diff
@@ -0,0 +1,36 @@
+# tmt run
+
+> `run` command is used to execute test steps. By default all test steps are run. See also: `tmt`, `tmt-try`.
+> More information: <https://tmt.readthedocs.io/en/stable/overview.html#run>.
+
+- Run all test steps for each plan:
+
+`tmt run`
+
+- Run selected plans and tests:
+
+`tmt run plan -n <plan_name> test -n <test_name>`
+
+- Show what tests a specific plan would run:
+
+`tmt run -vv discover plan -n <plan_name>`
+
+- Show test results from the last run in a web browser:
+
+`tmt run -l report --how html --open`
+
+- Add context to a test run:
+
+`tmt --context foo=bar --context baz=qux,quux run ...`
```

Comment:
What about using some tangible example? Perhaps `trigger=commit` or something similar?

Reply:
I'm ok with that, but fwiw, having foo=bar there tells people that the context keys can be custom. How about keeping first foo=bar and second some reserved dimension, like the mentioned trigger?

Reply:
Well I think the user would eventually have to lookup the fmf/tmt documentation to figure out how the list/dict etc. contexts are defined, or even when they need it in general. I think it's fine with either variant because for me the tl;dr signals that this is an important feature that the user should consider

Reply:
![image](https://github.com/user-attachments/assets/912a3844-9edf-4ea7-9e58-438575992f6d)
ok like this? foo being without color to show it's custom, while `distro` is reserved.

Reply:
Sounds good, thanks!